### DISCUSSION TITLE
Support for multiple cartography volumes.

### DISCUSSION THREAD

**Discussion by DanielWillett** [SmartlyDressedGames/Unturned-3.x-Community#4060] [Ideas]
Allow placing multiple cartography volumes and saving map images for each of them. Then in-game the displayed map depends on which volume you are in. Allow giving each volume a priority so overlapping volumes still work. Perhaps make one marked as primary so if you're not in any it shows that one.

**Comment by Jdance-Media** [comment]
This would be really cool for custom gamemodes. In the past I've used Insane maps to create multiple 'maps' for some FPS gamemodes, but the map itself showed every other portion of the Insane map.

**Reply by SDGNelson** [reply]
I would like to implement this, but no ETA.
